src/server.cpp

```diff
@@ -3154,7 +3154,7 @@ void Server::spawnParticle(const std::string &playername, v3f pos,
 			collision_removal, vertical, texture, animation, glow);
 }
 
-u32 Server::addParticleSpawner(u16 amount, float spawntime,
+s32 Server::addParticleSpawner(u16 amount, float spawntime,
```

Comment:
Inconsistent with the return value of `ServerEnvironment::addParticleSpawner` and ServerEnvironment::m_particle_spawners`